source/extensions/stat_sinks/metrics_service/grpc_metrics_service_impl.cc

```diff
@@ -113,19 +113,19 @@ void MetricsServiceSink::flush(Stats::MetricSnapshot& snapshot) {
                                             snapshot.histograms().size());
   for (const auto& counter : snapshot.counters()) {
     if (counter.counter_.get().used()) {
-      flushCounter(counter, snapshot.snapshotTimeMs());
+      flushCounter(counter, snapshot.snapshotTime().count());
```

Comment:
hoist the conversion to an integer number of milliseconds out of the loop, here and below.

Reply:
done